src/plugins/analysis/ipc/view/ipc_analyzer.html

```diff
@@ -0,0 +1,40 @@
+{% extends "analysis_plugins/general_information.html" %}
+
+{% block analysis_result_details %}
+
+<table class="table table-bordered">
+    <colgroup>
+        <col width="50">
+        <col width="150">
+        <col width="600">
+    </colgroup>
+    <tbody class="table-analysis">
+        {% for target in firmware.processed_analysis[selected_analysis]['full']['ipcCalls'].keys()|sort %}
```

Comment:
We can declutter the template code a bit by putting this inside a jinja variable:
```suggestion
        {% set ipc_calls = firmware.processed_analysis[selected_analysis]['full']['ipcCalls'] %}
        {% for target in ipc_calls.keys()|sort %}
```

Reply:
Thanks, using the jinja variable everywhere now.